azure-keyvault/azure/keyvault/secrets/_client.py

```diff
@@ -1,20 +1,40 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import functools
 import uuid
 
 from azure.core.configuration import Configuration
-from azure.core.pipeline.policies import UserAgentPolicy, HeadersPolicy, RetryPolicy, \
-    RedirectPolicy, CredentialsPolicy, HTTPPolicy
+from azure.core.pipeline.policies import (
+    UserAgentPolicy,
+    HeadersPolicy,
+    RetryPolicy,
+    RedirectPolicy,
+    CredentialsPolicy,
+    HTTPPolicy,
+)
 from azure.core.pipeline.transport import RequestsTransport, HttpRequest
 from azure.core.pipeline import Pipeline
 from azure.core.exceptions import ClientRequestError
 
 from msrest import Serializer, Deserializer
 
-from ._models import Secret, SecretPaged, SecretAttributes
+from ._models import (
+    DeletedSecret,
+    DeletedSecretPaged,
+    Secret,
+    SecretAttributesPaged,
+    SecretAttributes,
+    _SecretManagementAttributes,
+)
 
 from .._internal import _BackupResult
 
-class BearerTokenCredentialPolicy(HTTPPolicy):
 
+class BearerTokenCredentialPolicy(HTTPPolicy):
```

Comment:
We should make this _BearerTokenCredentialPolicy and move it to _internal.py for now.  Eventually this will be defined in azure.core.